unitTests/gtestwrapper.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
```

Comment:
Please use ordinary include guards, `#pragma once` is non-standard and while being accepted by gcc & clang, there is no guarantee for that.

Reply:
Good point. I tend to use this since most of the most popular compilers implement it (gcc, clang, visual studio, intel), but I did not consider that Exiv2 is used much more extensively than my personal projects